.github/workflows/sync-sf-translations.yml

```diff
@@ -0,0 +1,79 @@
+name: Sync Symfony translations
+
+on:
+  schedule:
+    - cron: '30 6,12,18 * * *'
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Generate
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
+          repository: symfony/symfony
+
+      - name: Get lowest version
+        id: version
+        run: |
+          WORKDIR=$(pwd)
+          
+          git clone https://github.com/symfony-tools/symfony-translation-checker source
+          cd $WORKDIR/source && composer install --no-interaction
+          
+          version=$(cd $WORKDIR/source && php bin/console app:lowest-supported-versions)
+
+          echo "::set-output name=lowest_branch::$version"
+          
+          rm -rf $WORKDIR/source
+
+      - name: Download dependencies
+        run: composer install --no-interaction --prefer-stable
+
+      - name: Generate data files
+        id: generate_data
+        run: |
+          version=${{ steps.version.outputs.lowest_branch }}
+          
+          git checkout $version
+          php .github/sync-translations.php
+          
+          echo "::set-output name=lowest_branch::$version"
```

Comment:
`set-output` is deprecated

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Thank you